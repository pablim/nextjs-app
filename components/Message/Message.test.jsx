import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Message } from '../'

describe('Reders tests', () => {
	test('Render Message component', () => {
		const { container } = render(<Message />);
		expect(container).toMatchSnapshot();
	});
})
