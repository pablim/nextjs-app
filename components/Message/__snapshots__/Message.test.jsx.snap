// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reders tests Render Message component 1`] = `<div />`;
